packages/back-end/src/api/features/updateFeature.ts

```diff
@@ -146,6 +146,7 @@ export const updateFeature = createApiRequestHandler(updateFeatureValidator)(
           environments: orgEnvs,
           publish: true,
           changes: revisionChanges,
+          org: req.organization,
```

Comment:
We will need some way to support a "bypass reviews" option for the REST API.  Otherwise, it's going to be really hard to automate things.  This is not a blocker for now, just something to keep in mind for the future.